packages/@aws-cdk/aws-route53/README.md

```diff
@@ -132,6 +132,26 @@ Constructs are available for A, AAAA, CAA, CNAME, MX, NS, SRV and TXT records.
 Use the `CaaAmazonRecord` construct to easily restrict certificate authorities
 allowed to issue certificates for a domain to Amazon only.
 
+### Working with existing record sets
+
+Use the `deleteExisting` prop to delete an existing record set before deploying the new one.
+This is useful if you want to minimize downtime and avoid "manual" actions while deploying a
+stack with a record set that already exists. This is typically the case for record sets that
+are not already "owned" by CloudFormation or "owned" by another stack or construct that is
+going to be deleted (migration).
+
+```ts
+declare const myZone: route53.HostedZone;
+
+new route53.ARecord(this, 'ARecord', {
+  zone: myZone,
+  target: route53.RecordTarget.fromIpAddresses('1.2.3.4', '5.6.7.8'),
+  deleteExisting: true,
```

Comment:
In the beginning I thought  that `upsert` would have been a good name for this prop, but it's not a real UPSERT operation?

Reply:
The existing name is definitely better.